sessions/006/README.md

```diff
@@ -326,3 +326,256 @@ pvc-0bd38196-2f5b-4a05-8917-b43bd2dde50b   20Gi       RWO            Delete
+	...
+	Topic: t0	Partition: 999	Leader: 1	Replicas: 0,1,2	Isr: 1,0,2
+
+kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --topic t1 --describe
+Topic: t1	TopicId: Zz3T1a_hQiGvHv8LLK7H9Q	PartitionCount: 1000	ReplicationFactor: 3	Configs: min.insync.replicas=2,message.format.version=3.0-IV1
+	Topic: t1	Partition: 0	Leader: 1	Replicas: 1,0,2	Isr: 1,0,2
+	Topic: t1	Partition: 1	Leader: 1	Replicas: 0,2,1	Isr: 1,0,2
+	Topic: t1	Partition: 2	Leader: 1	Replicas: 2,1,0	Isr: 1,0,2
+	Topic: t1	Partition: 3	Leader: 1	Replicas: 1,2,0	Isr: 1,0,2
+	...
+	Topic: t1	Partition: 999	Leader: 1	Replicas: 0,1,2	Isr: 1,0,2
+```
+We can also get the topic t2's information even though it caused disk full when creating topic t2.
+If this command didn't show expected results, try to run the command later since zookeeper also needs time to do data recovery.
+
+```
+kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --describe --topic t2
+Topic: t2	TopicId: LCYz44eWSYmMA1xhY9BpdQ	PartitionCount: 1000	ReplicationFactor: 3	Configs: min.insync.replicas=2,message.format.version=3.0-IV1
+	Topic: t2	Partition: 0	Leader: 0	Replicas: 1,0,2	Isr: 0,1,2
+	Topic: t2	Partition: 1	Leader: 0	Replicas: 0,2,1	Isr: 0,1,2
+	Topic: t2	Partition: 2	Leader: 0	Replicas: 2,1,0	Isr: 0,1,2
+	Topic: t2	Partition: 3	Leader: 0	Replicas: 1,2,0	Isr: 0,1,2
+	...
+	Topic: t2	Partition: 999	Leader: 0	Replicas: 1,2,0	Isr: 0,1,2
+```
+
+
+Finally, we delete the old volumes to reclaim some space, and optionally set the retain policy back to Delete on new volumes.
+
+```sh
+$ kubectl delete pv $(kubectl get pv | grep Available | awk '{print $1}')
+persistentvolume "pvc-04b55551-fe7f-4662-9955-5e4baaf4df57" deleted
+persistentvolume "pvc-18280833-16a8-4cd5-8c6f-eb764acd3ce9" deleted
+persistentvolume "pvc-a148ee8b-2eef-422b-a35e-b71714b1ef85" deleted
+
+$ kubectl patch pv $(kubectl get pv | grep "$CLUSTER_NAME-zookeeper" | awk '{print $1}') --type merge -p '
+    spec:
+      persistentVolumeReclaimPolicy: Delete'
+persistentvolume/pvc-0bd38196-2f5b-4a05-8917-b43bd2dde50b patched
+persistentvolume/pvc-1a66039a-14e7-4416-9319-6d6437543c02 patched
+persistentvolume/pvc-ca348d35-f466-446e-9f93-e3c15722d214 patched
+
+$ kubectl get pv | grep $CLUSTER_NAME-zookeeper
+pvc-0bd38196-2f5b-4a05-8917-b43bd2dde50b   20Gi       RWO            Delete           Bound    test/data-my-cluster-zookeeper-1                                         ocs-external-storagecluster-ceph-rbd            18m
+pvc-1a66039a-14e7-4416-9319-6d6437543c02   20Gi       RWO            Delete           Bound    test/data-my-cluster-zookeeper-0                                         ocs-external-storagecluster-ceph-rbd            18m
+pvc-ca348d35-f466-446e-9f93-e3c15722d214   20Gi       RWO            Delete           Bound    test/data-my-cluster-zookeeper-2                                         ocs-external-storagecluster-ceph-rbd            18m
+```
```

Comment:
These are the PVs from the Kafka example. Did you forget to update?

Reply:
No, I thought on one would notice it...:)